site/docs/v1/tech/themes/kickstart-custom-theme.adoc

```diff
@@ -0,0 +1,400 @@
+    }
+  ],
+  "requests":[
+    "&{json/createTheme.json}",
+    "&{json/updateTheme.json}",
+    "&{json/createApplication.json}",
+    "&{json/createUser.json}",
+    "&{json/registerUser.json}"
+  ]
+}
+```
+
+There are three sections in this code: `"variables"`, `"apiKeys"`, and `"requests"`.
+
+The `"variables"` section defines identifiers for the key components of our FusionAuth instance. In this section, `"apiKey"`, `"themeId"`, and `"userId"` are randomly generated UUIDs. We'll use the arbitrary values in `"applicationId"` and `"clientSecret"` later on.
+
+The `"apiKeys"` section defines the key through which our requests will be executed. At least one `"apiKey"` is required for every `kickstart.json` file.
+
+The `"requests"` section defines the API requests that perform our automations. Here, each request is encased in a JSON file, which we need to define separately. Let's do this now.
+
+=== The API Request JSON Files
+
+Create a subfolder in the `kickstart` folder called `json`. In the `json` folder, add a file called `createTheme.json` containing the following code:
+
+```json
+{
+  "method" : "POST",
+  "url" : "api/theme/#{themeID}",
+  "body" : {
+    "sourceThemeId" : "75a068fd-e94b-451a-9aeb-3ddb9a3b5987",
+    "theme" : {
+      "name" : "Dark Theme"
+    }
+  }
+}
+```
+
+This request creates the dark theme. It uses the `"sourceThemeId"` attribute to copy everything from the default FusionAuth theme, the Id of which is always `75a068fd-e94b-451a-9aeb-3ddb9a3b5987`. It also assigns the UUID initialized and contained in the `#{themeID}` variable as this theme's Id by setting it as the resource Id in the path of the URL.
+
+Create a file called `updateTheme.json` and add the following to it:
+
+```json
+{
+  "method" : "PATCH",
+  "url" : "api/theme/#{themeID}",
+  "body" : {
+    "theme" : {
+      "stylesheet" : "@{darkTheme.css}"
+    }
+  }
+}
+```
+
+This request applies our `darkTheme.css` stylesheet to the theme we created.
+
+Create a file called `createApplication.json` and copy the following into it:
+
+```json
+{
+  "method" : "POST",
+  "url" : "/api/application/#{applicationID}",
+  "body" : {
+    "application":{
+      "name" : "Kickstart App",
+      "oauthConfiguration" : {
+        "authorizedRedirectURLs" : [
+          "http://localhost:3000/oauth-redirect"
+        ],
+        "clientId" : "#{applicationID}",
+        "clientSecret" : "#{clientSecret}",
+        "logoutURL": "http://localhost:3000/logout",
+        "enabledGrants": [
+          "authorization_code",
+          "refresh_token"
+        ],
+        "requireRegistration" : "true"
+      }
+    }
+  }
+}
+```
+
+This request creates the application and configures its OAuth settings as they appear in the 5-minute guide.
+
+Create a file called `createUser.json` containing the following:
+
+```json
+{
+  "method": "POST",
+  "url": "/api/user/registration/#{userID}",
+  "body": {
+    "user": {
+      "email": "test_user@email.com",
```

Comment:
Can we make this `richard@example.com` and a password of `password` for consistency?